### DISCUSSION TITLE
Original SDRAM source code

### DISCUSSION THREAD

**Discussion by felipecgoliv** [UECIDE/UECIDE#159] [Q&A] [answered]
Hi,

In the pic32 implementation, retromaster's SDRAM code from https://retromaster.wordpress.com/pic32/interfacing-sdram-to-pic32/ was used. Do you have the original source code that he shared in his wordpress? The rapidshare link from his post is down.

**Comment by majenkotech** [comment, marked as answer]
Looks like we used that code in RetroBSD. A variant could be here: 

https://github.com/RetroBSD/retrobsd/blob/master/sys/pic32/sdram.S 

https://github.com/RetroBSD/retrobsd/blob/master/sys/pic32/sdram.h

**Reply by felipecgoliv** [reply]
That's it! Thanks a lot!
